src/openms/source/ANALYSIS/OPENSWATH/OpenSwathWorkflow.cpp

```diff
@@ -582,9 +581,7 @@ namespace OpenMS
             // Step 4: write all chromatograms and features out into an output object / file
             // (this needs to be done in a critical section since we only have one
             // output file and one output map).
-#ifdef _OPENMP
-#pragma omp critical (osw_write_out)
-#endif
+            #pragma omp critical (osw_write_out)
```

Comment:
if these are not needed... why did we have them in the first place?

Reply:
they are only needed if an omp function is used, such as `omp_get_num_threads()` (which is not defined if OMP is off). Precompiler directives do not require #idefs since they will be ignored if unknown.